minigui/minigui.ts

```diff
@@ -0,0 +1,436 @@
+import * as board from './board'
```

Comment:
You need the license headers in the .ts files.

Reply:
done